src/main/java/service/TicketService.java

```diff
@@ -1,88 +1,51 @@
 package service;
 
+import dao.TicketDaoImpl;
+import dao.UserDaoImpl;
 import entity.BaseEntity;
 import entity.Printable;
-import model.StadiumSector;
-import model.collection.list.CustomArrayList;
-import model.collection.set.CustomHashSet;
-import model.ticket.BusTicket;
-import model.ticket.CinemaTicket;
+import model.TicketType;
 import model.ticket.Ticket;
-import model.user.Admin;
-import model.user.Client;
 import model.user.User;
 
-import java.math.BigDecimal;
 import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.UUID;
 
 public class TicketService extends BaseEntity implements Printable {
 
     public static void main(String[] args) {
 
-        BigDecimal price = new BigDecimal("60.55");
-        Ticket ticket1 = new Ticket();
-        Ticket ticket2 = new Ticket("Opera", (short) 123, new Timestamp(System.currentTimeMillis()), false, StadiumSector.A, (float) 5.001, price);
-        Ticket ticket3 = new Ticket("Theatre", (short) 124, new Timestamp(System.currentTimeMillis()));
+        TicketDaoImpl ticketDao = new TicketDaoImpl();
+        UserDaoImpl userDao = new UserDaoImpl();
 
-        BaseEntity ticket = new Ticket();
-        System.out.println(ticket.getId());
-        ticket.setID("563gdj");
+        Ticket ticket = new Ticket(UUID.fromString("cb1565a0-68e2-48b6-a7d6-019d4935db43"), TicketType.DAY,
```

Comment:
if you define cascade dependency, then you need to create this record (User) before this one (Ticket) or create them during one transaction in the same time.

I recommend to refactor code into this class and change the order:
1. create users and add them to your storage
2. retrieve some user ids from db and create Tickets

Reply:
Created users and add them to your storage
Retrieved some user ids from db and created Tickets